pkg/maps/egressmap/policy.go

```diff
@@ -178,14 +177,12 @@ func (k *EgressPolicyKey4) Match(sourceIP netip.Addr, destCIDR netip.Prefix) boo
 
 // GetSourceIP returns the egress policy key's source IP.
 func (k *EgressPolicyKey4) GetSourceIP() netip.Addr {
-	addr, _ := netipx.FromStdIP(k.SourceIP.IP())
-	return addr
+	return k.SourceIP.Addr()
```

Comment:
Just to be clear, `FromStdIP` implicitly unmaps IPv6-mapped IPv4 addresses, but we don't need it because we don't accept such addresses here, right?

Reply:
Yeah I'm pretty sure we're ok on that front.